doc-content/jbpm-docs/src/main/asciidoc/BPMN2/copy-elements-proc.adoc

```diff
@@ -0,0 +1,19 @@
+[id='copy-elements']
+
+= Copying elements
+You can copy individual elements and finished business processes.
```

Comment:
IMO the word "finished" is not necessary, you can make a copy of a non finished process anyway.